src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +61,16 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    /**
+     * Comparator for sorting people by alphabetical order.
+
+    public static Comparator<Name> nameComparator = new Comparator<Name>() {
```

Comment:
Just a thought, how about overriding the CompareTo function instead? Since the names are compared with each other instead of the SortCommand using the Comparator to compare names. Maybe I missed something but why are there 2 comparators?